files/lpasspw

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo -n "$*: " >/dev/null
+answer=`cat $LPASS_HOME/pw`
```

Comment:
Yea the profiled is a bash script, so you could put some logic in there...
This would require a new shell after making the $LPASS_HOME/pw file, but thats probably OK

```bash
# pseudocode (not bash specific)
if $LPASS_HOME/pw
  export LASTPASS_ASKPASS = 
fi